usr.sbin/daemon/daemon.c

```diff
@@ -371,12 +377,12 @@ daemon_eventloop(struct daemon_state *state)
 	 */
 	(void)madvise(NULL, 0, MADV_PROTECT);
 
-	if (pipe(state->pipe_fd)) {
+	if (pipe(&state->pipe_rd)) {
```

Comment:
This may raise some eyebrows. We can always have a local int[2] to make is safer

Reply:
Yeah, no, that's just way too ick

Reply:
fixed with local variable